crates/iota-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -675,6 +677,10 @@ pub struct AuthorityEpochTables {
 
     /// Holds the timestamp of the most recently generated round of randomness.
     pub(crate) randomness_last_round_timestamp: DBMap<u64, CommitTimestampMs>,
+
```

Comment:
```suggestion

    // Tables for recording per-object debts for congestion control.

    //
```
There are quite a few tables just above that, labeled as `Tables for recording state for RandomnessManager`. Add this separator to make it clear these two new tables are not related to randomness manager.